### ISSUE TITLE
Horizontal Gridlines in Animation do not scale 

### ISSUE THREAD

**Issue by JoostZ** [JoostZ/ShowRow#66]
The last changes with #19 has broken this scaling.

**Comment by Tomhz** [comment]
I don't see any change in 1.0.6532

![rowanimation_9](https://user-images.githubusercontent.com/5354652/32989484-4c1d41fa-cd17-11e7-96a6-6f9c5d8dcb1d.png)

**Comment by JoostZ** [comment]
This is in a different issue. Here I was talking about the horizontal lines in the Animation window. I fixed this before removing the grid completely.

**Comment by Tomhz** [comment]
Yes, I should have been more clear that this bug is about the Profile Window. I will change the Issue description

**Comment by JoostZ** [comment]
No it is not. I created this issue when I fixed the scaling problem with the handposition display. There is a different issue for the scaling of the profile.

**Comment by Tomhz** [comment]
Oh, I am afraid I am messing up now. ;-(

**Comment by JoostZ** [comment]
I changed the title back. The issue you talked about is #40. It was not in the project but I added it now.

This issue should be closed now, right?

**Comment by Tomhz** [comment]
This issue is about the horizontal scaling in Profile?

**Comment by JoostZ** [comment]
No, it is about the horizontal lines in the Animation window. When I fixed the slow resizing with the hand position track I made a mistake and these horizontal grid lines did not resize when resizing the window. So I created this as a bug. Then we decided that we did not want these horizontal lines, so I removed them (with #68. So the scaling described is no longer relevant.

I now see that the title is confusing. It should mention Animation, not RowAnimation (which is more general).

I will rename the title again.
